src/js/view/controls/components/simple-tooltip.js

```diff
@@ -29,13 +40,11 @@ export function SimpleTooltip(attachToElement, name, text, openCallback) {
         }
     };
 
-    if ('PointerEvent' in window) {
-        attachToElement.addEventListener('pointerover', instance.open);
-        attachToElement.addEventListener('pointerout', instance.close);
-    } else {
-        attachToElement.addEventListener('mouseover', instance.open);
-        attachToElement.addEventListener('mouseout', instance.close);
-    }
+    attachToElement.addEventListener('mouseover', instance.open);
+    attachToElement.addEventListener('mouseout', instance.close);
+    attachToElement.addEventListener('touchstart', (evt) => {
+        instance.pointerType = getPointerType(evt);
```

Comment:
I think it's better to do `instance.touch` rather than check `instance.pointerType === 'touch'`

Reply:
coolio